recipes/mesa/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "mesa" %}
```

Comment:
I'm not sure if the name should be mesa, as this does not build the whole project.
In case we change the name, we should be careful to not generate another ```mesa-*``` package which overwrites generates the same files (libGL.so and friends are just being installed in $PREFIX/lib on Linux).